packages/vaex-core/setup.py

```diff
@@ -23,20 +23,16 @@
 license = 'MIT'
 version = version.__version__
 url = 'https://www.github.com/maartenbreddels/vaex'
-# TODO: can we do without requests and progressbar2?
 # TODO: after python2 supports frops, future and futures can also be dropped
-# TODO: would be nice to have astropy only as dep in vaex-astro
-setup_requires = ["numpy~=1.16"]
-install_requires_core = ["numpy~=1.16", "aplus", "tabulate>=0.8.3",
-                         "future>=0.15.2", "pyyaml", "progressbar2",
-                         "requests", "six", "cloudpickle", "pandas", "dask!=2022.4.0",
-                         "nest-asyncio>=1.3.3", "pyarrow>=5.0.0", "frozendict!=2.2.0",
+setup_requires = ["numpy~=1.17"]
+install_requires_core = ["numpy~=1.17", "aplus", "tabulate>=0.8.3", "dask!=2022.4.0",
+                         "future>=0.15.2", "pyyaml", "six", "cloudpickle", "pandas",
```

Comment:
python 3.12 support was introduced in pandas 2.1, so that's the implicit minimum version CI needs to support. e.g. the error with `'M'` as period was already changed in pandas v1.0. So I think vaex CI was construed with pandas v0.

Reply:
https://github.com/ddelange/vaex/pull/3/commits/68e689b8ad0215545fc456d15911cf7bea562ec8